docs/releases.md

```diff
@@ -30,14 +30,10 @@ To generate the next release binary:
     - each release includes the sub headings "What's New", "What's Fixed", and "Known Issues"
     - consult with Product Management on wording if needed
 2. Commit and ultimately merge to `master` branch
-3. Merge the `master` branch into `production` branch and push to GitHub:
-    - `git checkout master`
-    - `git pull upstream master` (to make sure you have the latest)
-    - `git checkout production`
-    - `git pull upstream production` (to make sure you have the latest)
-    - `git merge master`
-    - `git push upstream production`
-    - Test Pilot's Jenkins will now build and sign the extension (see ["Extension Signing"](#extension-signing))
+3. Create a pull request comparing the `master` branch and `production` branch on GitHub
```

Comment:
This PR is to be _from_ `master` _to_ `production`, correct?  If so, it should be explicitly stated.

Reply:
>This PR is to be from master to production, correct?

Yup 👌 I clarified and added a link ☑️